docs/tutorials/ad-hoc-simulation/planning-simulation.md

```diff
@@ -224,7 +224,7 @@ When driving across a crosswalk, ego vehicle will respond to objects on the cros
 
 #### Unsignalized crosswalk
 
-To pass through an unsignalized crosswalk, the ego vehicle will first slow down and stop for a safety check. Once safety is confirmed, the vehicle will proceed.
+To pass through an unsignalized crosswalk, if the crosswalk is free of obstacles, the ego vehicle will cross it without slowing down. However, if obstacles are present, the ego vehicle will slow down and stop for a safety check. Once it's safe to proceed, the ego vehicle will continue its journey.
```

Comment:
It will be more clear if you follow the words on the official document.
- According to [the document](https://autowarefoundation.github.io/autoware.universe/main/planning/behavior_velocity_crosswalk_module/), we say target objects instead of obstacles.
- According to the document, we say yield decision instead of safety check.